tests/pytest/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-django
+pytest-mock
```

Comment:
Nice. I had only only quickly glanced at it and missed the "undoing the mocking automatically after the end of the test" part, which is useful.